Kernel/Thread.cpp

```diff
@@ -786,18 +789,16 @@ static DefaultSignalAction default_signal_action(u8 signal)
 bool Thread::should_ignore_signal(u8 signal) const
 {
     VERIFY(signal < 32);
-    auto& action = m_signal_action_data[signal];
+    auto const& action = m_signal_action_data[signal];
     if (action.handler_or_sigaction.is_null())
         return default_signal_action(signal) == DefaultSignalAction::Ignore;
-    if ((sighandler_t)action.handler_or_sigaction.get() == SIG_IGN)
```

Comment:
if (true) return true; return false... Lovely, haven't seen one of these in a while :-)